### DISCUSSION TITLE
I can't get my azuraCast update from inside the admin panel

### DISCUSSION THREAD

**Discussion by Sweetness2k** [AzuraCast/AzuraCast#6227] [General]
Hello am trying to update my azuracast from in the panel and it keep giving me a error..

ConnectException at /var/azuracast/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php L210: cURL error 6: Could not resolve host: updater (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://updater:8080/v1/update

**Comment by Sweetness2k** [comment]
we get this issue fix...

**Reply by Sakretsos** [reply]
Hello @Sweetness2k, can you please tell us how you manage to fix the above issue?
Thanks in advance!

**Reply by Sweetness2k** [reply]
i done nothing it just start working on it's own and now today i have the same issue again can't update from inside the panel..

**Comment by Sweetness2k** [comment]
Hello i can't update my latest version today because am getting another error like few months ago...

`ConnectException at /var/azuracast/www/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php L210: cURL error 6: Could not resolve host: updater (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://updater:8080/v1/update`

**Reply by BusterNeece** [reply]
This seems to happen a lot with our little web updater utility container. It likes to shut down and then disable web updates afterward.

There are two ways around this:
- SSH into your machine, and run:
```bash
cd /var/azuracast
docker-compose restart updater
```

- Or, while you're SSH'ed into the machine, just run the standard update process using:
```bash
cd /var/azuracast
./docker.sh update
```

**Reply by Sweetness2k** [reply]
thank you for your reply, every thing is all updated now...
